src/index.ts

```diff
@@ -106,6 +114,12 @@ function activate(
         createGitMenu(app, gitExtension, factory.defaultBrowser, settings),
         { rank: 60 }
       );
+
+      // Listen for changes to extension settings:
+      settings.changed.connect((settings: ISettingRegistry.ISettings) => {
+        gitExtension.refreshInterval = settings.composite
+          .refreshInterval as number;
+      });
```

Comment:
It looks like you'll need more than this to ensure that the `refreshInterval`'s in the model instance and the settings are in sync when jlab starts up/is refreshed.

Reply:
@telamonian Thanks! I added logic for syncing the refresh interval as soon as the settings are resolved which should cover the start-up and refresh cases mentioned below.